cloud_testing/xctest_parser/src/main/kotlin/xctest/Xctestrun.kt

```diff
@@ -87,14 +88,26 @@ object Xctestrun {
         return result
     }
 
+    private fun NSDictionary.deepClone() : NSDictionary {
+        val newDictionary = NSDictionary()
+
+        this.forEach { (key, value) ->
+            val newValue = NSObject.fromJavaObject(value.toJavaObject())
```

Comment:
ok, the to/from JavaObject api does makes this easy. It also looks like we probably don't need to do this forEach for the top level dictionary, we could just have the entire method be `return NSObject.fromJavaObject(this.toJavaObject())`, right?